ui/window.go

```diff
@@ -155,9 +158,30 @@ func (win *Window) NewLoad() (*load.Load, error) {
 		}
 	}
 
+	lang := values.DefaultLanguage
+	if mw != nil && mw.IsAssetManagerDB() {
+		lang = mw.GetLanguagePreference()
+	}
+
+	var err error
+	l.DexCore, err = dexcore.New(l.WL.Wallet.Root, lang, l.WL.Wallet.Net) // TODO: We should be able to update dex core's language when the user changes language.
+	if err != nil {
+		return nil, fmt.Errorf("Error initializing dex core: %v", err)
+	}
+
 	return l, nil
 }
 
+// RunDexCore starts a goroutine that starts/initializes dex core. It blocks
+// until dex core is ready.
+func (win *Window) RunDexCore(ctx context.Context) {
+	go func() {
+		win.load.DexCore.Run(ctx)
+		win.Quit <- struct{}{} // in the event that Run returns prematurely prior to context cancellation
+	}()
+	<-win.load.DexCore.Ready()
```

Comment:
This will have more use in the DEX page, can show a loader while DEX is initializing and remove the loader when dexCore is ready.